GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -210,7 +210,9 @@ public static class VerbParameters
         {
             public static class Mount
             {
-                public const string ServiceName = "internal_use_only_service_name";
+                public const string InternalParameters = "internal_use_only_parameters";

```

Comment:
Minor, but I would name it just "internal_use_only" and drop the "_parameters". Or even just "internal_use". Make the CLI, especially when captured in Task Manager, etc, slightly less verbose.